tidb-cloud/serverless-tier-limitations.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Serverless Tier Limitations
+summary: Learn about the limitations of TiDB Cloud Serverless Tier.
+---
+
+# Serverless Tier Limitations
+
+<!-- markdownlint-disable MD026 -->
+
+Serverless Tier has some limitations below. We are also constantly filling in the feature gaps between Serverless Tier and Dedicated Tier. If you require these features or capabilities, please use [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) or [contact us](https://www.pingcap.com/contact-us/?from=en) for a feature request.
+
+## General Limitations
+
+- For each TiDB Cloud account, you can create one complimentary Serverless Tier cluster to use during the beta phase. You need to delete the existing Serverless Tier cluster then create a new one. 
+- Each Serverless Tier cluster has the following limitations:
+    - The storage size is limited to 5GiB (logical size) of OLTP storage and 5GiB of OLAP storage.
+    - The compute resource is limited to 1 vCPU and 1GiB RAM.
+    - The data transfer is limited to a total of 20 GiB in and out per week. If the 20 GiB limit is reached, the network traffic will be throttled to 10 KB/s.
+
+## Connection
+
+- Only [Standard Connection](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) can be used. You cannot use [Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections.md) or [VPC Peering](/tidb-cloud/set-up-vpc-peering-connections.md) to connect to Serverless Tier clusters. 
+- No "IP Access List" support.
```

Comment:
```suggestion
- IP Access List is not supported, which means you cannot specify which IP addresses are allowed to connect to your cluster.
```

Reply:
I think a simple description is ok. Don't make people consider a lot here.